robot/Cumulus/tests/browser/recurring_donations/create_a_fixed_length_recurring_donation_multiply_by.robot

```diff
@@ -0,0 +1,85 @@
+    ...                          select=Id
+    ...                          npe03__Recurring_Donation__c=&{recurringdonation}[Id]
+    ...                          ${ns}Recurring_Donation_Installment_Name__c=(1 of 50)
+
+    Go To Record Home            ${opportunity1}[0][Id]
+    Click Link                   link=Edit
+    Click Dropdown               Stage
+    Click Link                   link=Closed Won
+    Click Modal Button           Save
+
+    #Find 2nd Opportunity for Recurring Donation and Close It
+    @{opportunity2} =            Salesforce Query             Opportunity
+    ...                          select=Id
+    ...                          npe03__Recurring_Donation__c=&{recurringdonation}[Id]
+    ...                          ${ns}Recurring_Donation_Installment_Name__c=(2 of 50)
+
+    Go To Record Home            ${opportunity2}[0][Id]
+    Click Link                   link=Edit
+    Click Dropdown               Stage
+    Click Link                   link=Closed Won
+    Click Modal Button           Save
+
+    #Open NPSP Settings and run Rollups Donations Batch job
+    Run Donations Batch Process
+
+    #Check if 50th Opportunity for Recurring Donation Exists
+    @{opportunity} =             Salesforce Query             Opportunity
+    ...                          select=Id
+    ...                          npe03__Recurring_Donation__c=&{recurringdonation}[Id]
+    ...                          ${ns}Recurring_Donation_Installment_Name__c=(50 of 50)
+    Go To Record Home            ${opportunity}[0][Id]
+
+    #Check Rollups on Recurring Donation
+    Go To Record Home            &{recurringdonation}[Id]
+    Confirm Value                Total Paid Installments      2         Y
+    Confirm Value                Paid Amount                  $20.00    Y
+
+    #Check Soft Credit Rollups on Household Contact
+    Go To Record Home            &{contact2}[Id]
+    Select Tab                   Details
+    Scroll Element Into View     text:Household Donation Info
+    Confirm Value                Soft Credit Total            $20.00    Y
+    Confirm Value                Number of Soft Credits       2         Y
+
+    #Check Rollups on Recurring Account
+    @{account} =                 Salesforce Query             Account
+    ...                          select=Id
+    ...                          npe01__One2OneContact__c=&{contact}[Id]
+    Go To Record Home            ${account}[0][Id]
+    Select Tab                   Details
+    Scroll Element Into View     text:Membership Information
+    Confirm Value                Total Gifts                  $20.00    Y
+    Confirm Value                Total Number of Gifts        2         Y
+
+    #Cleanup Closed Won Opportunities to assist Suite Teardown
+    Salesforce Delete            Opportunity              ${opportunity1}[0][Id]
+    Salesforce Delete            Opportunity              ${opportunity2}[0][Id]
```

Comment:
Suggestion, instead of deleting like this, u can store the session record while u r on the record and suite teardown will take care of deleting records.

Reply:
@skristem thanks for that info... can you clarify what you mean by `storing a session record`? As far as I can tell it is being stored as a variable previously in the test.

Reply:
@skristem added `Store Session Record` keywords in for opportunities on tests